MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,209 @@
+    groupItem.Items.Add(new DataFormMultilineItem { FieldName = "Street" });
+    groupItem.Items.Add(new DataFormTextItem { FieldName = "State" });
+    groupItem.Items.Add(new DataFormTextItem { FieldName = "ZipCode" });
+    groupItem.Items.Add(new DataFormAutoCompleteItem { FieldName = "Country", ItemsSource = contactInfoViewModel.CountryNames }); 
+        items.Add(groupItem);
+
+    dataForm.AutoGenerateItems = false; 
+    dataForm.Items = items;
+
+{% endhighlight}   
+{% endtabs %} 
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class DataFormViewModel
+    {
+        public DataFormViewModel()
+        {
+            this.ContactDetails = new ContactInfo();
+            this.CountryNames = new List<string>
+            {
+                "United States",
+                "United Kingdom",
+                "France",
+                "Belgium",
+                "Germany"
+            };
+        }
+
+        public ContactInfo ContactDetails { get; set; }
+
+        public IList<string> CountryNames { get; set; }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class ContactInfo
+    {
+        public string Name { get; set; }
+
+        public string Password { get; set; }
+
+        public string Street { get; set; }
+
+        public string City { get; set; }
+
+        public string ZipCode { get; set; }
+
+        public string Country { get; set; }
+
+        public string Email { get; set; }
+    }
+
+{% endhighlight %}
+
+## Dynamically add DataForm items
+You can dynamically add the items using `Items` property of `SfDataForm`.
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.Items.Add(new DataFormTextItem() { FieldName = "State" });
+    dataForm.Items.Add(new DataFormTextItem() {FieldName = "Country" });
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Dynamically remove manually added dataform items
+You can dynamically remove the `DataFormViewItem` from the `Items` property of `SfDataForm`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.Items.RemoveAt(2);
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Dynamically clear manually added dataform items
+You can dynamically clear all the item views from `SfDataForm`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.Items.Clear();
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Dynamically reset manual dataform items
+You can reset the `SfDataForm.Items` and we can reset two or more items at a time.
+
+{% tabs %}
+{% highlight C# %}
+
+    DataFormTextItem textItem = new DataFormTextItem() { FieldName = "Age" };
+    dataForm.Items[2] = textItem;
+    
+    
+    DataFormDateItem dateItem = new DataFormDateItem() { FieldName ="Date Of Birth", RowOrder = 2, ItemsOrderInRow = 1, ColumnSpan = 3 });
+
+    dataForm.Items[3] = dateItem;
+    
+{% endhighlight %}
+{% endtabs %}
+
+
+## Dynamically add Dataform group items
+You can dynamically add custom group items using `Items` property of `SfDataForm`.
```

Comment:
You can dynamically add custom group editor by  using  the `Items` property of `SfDataForm`

Reply:
content updated